src/game_event.cpp

```diff
@@ -55,6 +55,20 @@ void Game_Event::SetSaveData(lcf::rpg::SaveMapEvent save)
 	data()->ID = event->ID;
 	SetMapId(map_id);
 
+	// Sanity checks on bad save data
+	auto checkMoveRoute = [&](const auto& mr, auto& idx, const auto& name) {
+		const auto n = static_cast<int>(mr.move_commands.size());
+		if (idx < 0 || idx > n) {
+			idx = n;
+			Output::Warning("Event {}: Save Data invalid {}={}. Fixing ...", data()->ID, name, idx);
```

Comment:
The warning should mention that it is a MoveRoute

Reply:
The lambda gets passed in `move_route_index` or `original_move_route_index` string name